### DISCUSSION TITLE
Usage on NvChad

### DISCUSSION THREAD

**Discussion by carloshnp** [iurimateus/luasnip-latex-snippets.nvim#18] [Q&A]
My nvim isn't recognizing it, nor is it in the LuaSnipListAvaliable, I guess I'm not setting it up properly in NvChad. Here is where I tried to set it up so far:

![image](https://github.com/iurimateus/luasnip-latex-snippets.nvim/assets/75504798/0dd9ac8a-2d00-4f1b-891f-22344f1d54ba)

How should I add it so it works no NvChad?

**Comment by wilville** [comment]
> My nvim isn't recognizing it, nor is it in the LuaSnipListAvaliable, I guess I'm not setting it up properly in NvChad. Here is where I tried to set it up so far:
> 
> ![image](https://private-user-images.githubusercontent.com/75504798/269090346-0dd9ac8a-2d00-4f1b-891f-22344f1d54ba.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyMzQzNTcsIm5iZiI6MTcxMzIzNDA1NywicGF0aCI6Ii83NTUwNDc5OC8yNjkwOTAzNDYtMGRkOWFjOGEtMmQwMC00ZjFiLTg5MWYtMjIzNDRmMWQ1NGJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDAyMjA1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMzNmIwZjUzM2MxN2MxMDI2YmVhMTRhNmE3YTk4ZDYyMjg0ZjZhMjgyNzE1MmZlNWYxM2NhMDE5NGZjMGFiNGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vWauIuwPnktx2YsqjPNpY3IUOpf-fP-VR8Oes05ZXkk)
> 
> How should I add it so it works no NvChad?

i struggled for ages as well to get it working lol here  is my config , you have to disable lazy load for some reason (I think because its made for packer) 
![image](https://github.com/iurimateus/luasnip-latex-snippets.nvim/assets/78002569/fc713b92-8b65-4642-a632-81a2fa33fe7a)

**Reply by iurimateus** [reply]
Correct, do not lazy load. On 'setup', it creates a FileType aucmd for 'tex' and 'markdown' (each).
